instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './CommentSection.css'
+import PropTypes from 'prop-types';
+import CommentForm from '../CommentForm';
+// import Comment from '../c';
+
+// import PostContainer from './PostContainer/PostContainer';
+
+class CommentSection extends React.Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            comments: props.comments
+        }
+    }
+
+addNewComment = (event, indexNum, commentText) => {
+    console.log(this.state.comments)
+    const copiedComments = this.state.comments
```

Comment:
This is technically not copying the comments - it is actually creating a reference in memory to the comments array. To copy the array, you could set copiedComments equal to `...this.state.comments`.